src/util.js

```diff
@@ -141,6 +142,26 @@ function resolveParameterDiff(params, defaults) {
   return Object.assign(defaults, params);
 }
 
+/**
+ * format log data into apache access log format
+ *
+ * @param {object} requests array of objects containing log data
+ *
+ * returns object in "%h %l %u %t \"%r\" %>s %b" format
+ */
+function formatObject(requests) {
```

Comment:
maybe call it: `accessLogFormat` and only handle 1 object, 
and then use `const result = requests.map(utils.accessLogFormat)`